exporters/otlp/testing-internal/src/main/java/io/opentelemetry/exporter/otlp/testing/internal/AbstractGrpcTelemetryExporterTest.java

```diff
@@ -170,6 +170,24 @@ protected AbstractGrpcTelemetryExporterTest(String type, U resourceTelemetryInst
   @BeforeAll
   void setUp() {
     exporter = exporterBuilder().setEndpoint(server.httpUri().toString()).build();
+
+    // Sanity check that TLS files are in PEM format.
+    assertThat(certificate.certificateFile())
```

Comment:
The issue is that the test encoded the key before it sets to the client, but the sdk read the key file and set to the client without encoding.

Reply:
Thanks a lot for clarifying this!